app/services/concerns/webhooks/receipt_handler.rb

```diff
@@ -0,0 +1,41 @@
+module Webhooks
+  module ReceiptHandler
+    def with_webhook_receipt(resource_model:, resource_id:, timestamp:, webhook_endpoint_id: nil)
+      # france travail webhooks are not linked to a webhook_endpoint
+      return if old_update?(resource_model: resource_model, resource_id: resource_id, timestamp: timestamp,
+                            webhook_endpoint_id: webhook_endpoint_id)
+
+      yield if block_given?
```

Comment:
Je pense pas qu'on veuille l'appeler sans block
```suggestion
      yield 
```